### DISCUSSION TITLE
how could i force update my bot's friendsList

### DISCUSSION THREAD

**Discussion by 1334310385** [wechaty/wechaty#2110] [General]
I have add a new friend,but i can't sent message to her,cause i can't find her buy name

my code:
      let solverContact = await global.bot.Contact.find({ alias: 'aaa' });
      if (!solverContact) {
        solverContact = await global.bot.Contact.find({ name: 'aaa' });
      }
  await solverContact.say('hell');

please tell what should i do?thanks

**Comment by huan** [comment]
Have you tried that let your friend send a message to you first? Then I believe you can find your friend by `Contact.find()`
